### ISSUE TITLE
Drag and drop is crashing app

### ISSUE THREAD

**Issue by inverce** [jaychang0917/SimpleRecyclerView#4]
When making small drag and drops on demo app one after another, very quickly, activity crashes and is restarted by ActivityManager

```
02-08 15:54:15.986 10754-26023/? E/JavaBinder: !!! FAILED BINDER TRANSACTION !!!
02-08 15:54:15.996 10754-26023/? I/ActivityManager: Restarting because process died: ActivityRecord{120ab7db u0 com.jaychang.demo.srv/.MainActivity t3043}
02-08 15:54:15.996 10754-26023/? E/JavaBinder: !!! FAILED BINDER TRANSACTION !!!
02-08 15:54:15.996 10754-26023/? W/ActivityManager: Exception when starting activity com.jaychang.demo.srv/.MainActivity
                                                    android.os.TransactionTooLargeException
                                                        at android.os.BinderProxy.transactNative(Native Method)
                                                        at android.os.BinderProxy.transact(Binder.java:496)
                                                        at android.app.ApplicationThreadProxy.scheduleLaunchActivity(ApplicationThreadNative.java:814)
                                                        at com.android.server.am.ActivityStackSupervisor.realStartActivityLocked(ActivityStackSupervisor.java:1301)
                                                        at com.android.server.am.ActivityStackSupervisor.startSpecificActivityLocked(ActivityStackSupervisor.java:1407)
                                                        at com.android.server.am.ActivityStack.resumeTopActivityInnerLocked(ActivityStack.java:1978)
                                                        at com.android.server.am.ActivityStack.resumeTopActivityLocked(ActivityStack.java:1536)
                                                        at com.android.server.am.ActivityStack.resumeTopActivityLocked(ActivityStack.java:1523)
                                                        at com.android.server.am.ActivityStack.startPausingLocked(ActivityStack.java:960)
                                                        at com.android.server.am.ActivityStack.finishActivityLocked(ActivityStack.java:2881)
                                                        at com.android.server.am.ActivityStack.finishTopRunningActivityLocked(ActivityStack.java:2738)
                                                        at com.android.server.am.ActivityStackSupervisor.finishTopRunningActivityLocked(ActivityStackSupervisor.java:2744)
                                                        at com.android.server.am.ActivityManagerService.handleAppCrashLocked(ActivityManagerService.java:13743)
                                                        at com.android.server.am.ActivityManagerService.makeAppCrashingLocked(ActivityManagerService.java:13640)
                                                        at com.android.server.am.ActivityManagerService.crashApplication(ActivityManagerService.java:14352)
                                                        at com.android.server.am.ActivityManagerService.handleApplicationCrashInner(ActivityManagerService.java:13868)
                                                        at com.android.server.am.NativeCrashListener$NativeCrashReporter.run(NativeCrashListener.java:86)
```

**Comment by jaychang0917** [comment]
Hi @inverce , is this issue reproducable?

**Comment by inverce** [comment]
Yes, and sadly quite easily, 
I have Samsung Grand Prime (5.0.2).

It only takes few seconds. Try making small and short gestures over drag icon, (those do not need to actually move an alement). It work especially well with draging diagonally.

**Comment by jaychang0917** [comment]
I cant reproduce in my device and emulator. And that is all the crash log?

**Comment by inverce** [comment]
I made video showcasing issue, and yes that is all the log i can see.

https://drive.google.com/file/d/0B7rAyBh5oQ2lNC1TUVM1dGU3clk/view?usp=sharing

**Comment by jaychang0917** [comment]
Thanks your report. Issue should be fixed in new release `1.0.3`.
